server/views/pages/view/cases.njk

```diff
@@ -73,7 +73,7 @@
         {
             attributes: {
                 id: 'com-' + loop.index,
-                "data-sort-value": case.probationPractitioner
+                "data-sort-value": case.probationPractitioner.name
```

Comment:
This was breaking sort on PP as this would return `[Object object]` for everyone